send/generic_send

```diff
@@ -176,7 +176,10 @@ if [ $ERR_CODE -eq 124 ]; then
 else
 	if [ "$DESTINATION_TYPE" = "$DESTINATION_TYPE_URL" ]; then
 		#the return code from curl is saved in STDOUT
-		if [ 200 -ne "$STDOUT" ]; then
+		if [ "000" = "$STDOUT" ]; then
+			ERR_CODE=28
+			echo "Curl has ended with error code 28 (timeout)!" >&2
```

Comment:
Unfortunately, you cannot be sure that "000" in $STDOUT means that curl ended with error code 28. "000" means basically, any client-side error.

Best solution would be to check $ERR_CORE and only if it's zero then check $STDOUT. 

Also, I would recommend to add `--show-error` to curl options which should produce error message from curl in case any error will occur.

Reply:
Changed to cover all possibilities.